src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,54 @@
-import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+import { Movie } from '../../types/Movie';
+
+interface MovieCardProps {
+  movie: Movie
+}
+
+export const MovieCard: React.FC<MovieCardProps> = ({
+  movie: {
+    title, description, imgUrl, imdbUrl,
+  },
+}) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={imgUrl}
+          alt="Film logo"
```

Comment:
```suggestion
          alt="Movie logo"
```
lets change it for consistency :)

Reply:
that's just how it was in the original file ¯\_(ツ)_/¯